src/metabase/api/collection.clj

```diff
@@ -70,40 +69,37 @@
   By default, admin users will see all collections. To hide other user's collections pass in
   `?exclude-other-user-collections=true`."
   [archived exclude-other-user-collections namespace]
-  {archived                       (s/maybe su/BooleanString)
-   exclude-other-user-collections (s/maybe su/BooleanString)
-   namespace (s/maybe su/NonBlankString)}
-  (let [archived? (Boolean/parseBoolean archived)
-        exclude-other-user-collections? (Boolean/parseBoolean exclude-other-user-collections)]
```

Comment:
This block "changed" because I removed this let, since we don't have to `parseBoolean` anymore.